frontend/web/components/App.js

```diff
@@ -443,64 +553,162 @@ const App = class extends Component {
                             <div />
                           )}
                         </Flex>
-                      </nav>
+                      </div>
                     )}
-                  {!isHomepage && (
-                    <Aside
-                      projectId={projectId || lastProjectId}
-                      environmentId={environmentId || lastEnvironmentId}
-                      toggleAside={this.toggleAside}
-                      asideIsVisible={asideIsVisible}
-                      disabled={!pageHasAside}
-                    />
-                  )}
-                  {isMobile && pageHasAside && asideIsVisible ? null : (
-                    <div>
-                      <ButterBar
-                        projectId={projectId}
-                        billingStatus={
-                          AccountStore.getOrganisation()?.subscription
-                            .billing_status
-                        }
-                      />
-                      {projectNotLoaded ? (
-                        <div className='text-center'>
-                          <Loader />
-                        </div>
-                      ) : (
-                        <Fragment>
-                          {user && (
-                            <OrganisationLimit
-                              id={AccountStore.getOrganisation()?.id}
-                            />
+                  {!isOrganisationSelect && !isCreateOrganisation && (
+                    <div className='py-0 bg-faint gap-4 d-flex px-3'>
+                      {activeProject ? (
+                        <>
+                          <NavSubLink
+                            icon={gitBranch}
+                            className={environmentId ? 'active' : ''}
+                            id={`features-link`}
+                            to={`/project/${projectId}/environment/${
+                              lastEnvironmentId || environmentId
+                            }/features`}
+                          >
+                            Environments
+                          </NavSubLink>
+                          <NavSubLink
+                            icon={<SegmentsIcon />}
+                            id={`segments-link`}
+                            to={`/project/${projectId}/segments`}
+                          >
```

Comment:
I think this is a great opportunity to clearly show in the UI/UX that a feature is a first-class citizen in our app, and belongs to Project level, we could easily do that by showing another tab called Features beside Environments showing exactly what we show today but without the ability to override the feature for an Environment, or even just asking for the environment if the user wants to do it. To distinguish the `features` option under the `Environments` tab we could name the option `Environment Features`. I think redundancy in this proposal worths it to gain clarity.

Reply:
Yeah this will clear the way for deciding how we have a separate screen for features, though it won't be part of this PR

Reply:
Resolving since as discussed internally this is out of the scope of this PR and will be addressed in the future.